docs/api.md

```diff
@@ -21,24 +21,30 @@
 - Implementation of request and response types can be found in
   [webapi/types.go](../webapi/types.go).
 
+- The initial version of the vspd API is numbered 3. This is because the first
+  version of the vspd API actually represents the third iteration of VSP APIs.
+  The first and second iterations of VSP API were implemented by
+  [dcrstakepool](https://github.org/decred/dcrstakepool).
```

Comment:
github.com

Reply:
ffs, second time I've made that mistake recently. I've somehow gotten `github.org` lodged into muscle memory. I've fixed that, and rebased onto master.